index.js

```diff
@@ -2159,4 +2187,5 @@
 //. [Ramda]:        http://ramdajs.com/
 //. [RegExp]:       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
 //. [Semigroup]:    https://github.com/fantasyland/fantasy-land#semigroup
+//. [Traversable]:  https://github.com/fantasyland/fantasy-land#traversable
```

Comment:
Can we remove the link definition for `[Foldable]`? I believe we're removing the only occurrence.